### PR TITLE
AC-215 - Apply MVP - VisitDashboardActivity

### PR DESCRIPTION
**Pull request by tmarzeion**
Done

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/visitdashboard/VisitDashboardActivity.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+        Intent intent = getIntent();
+
+        // Create fragment
+        VisitDashboardFragment visitDashboardFragment =
+                (VisitDashboardFragment) getSupportFragmentManager().findFragmentById(R.id.visitDashboardContentFrame);
+        if (visitDashboardFragment == null) {
+            visitDashboardFragment = VisitDashboardFragment.newInstance();
+        }
+        addFragmentToActivity(getSupportFragmentManager(),
```
Comment (AdamGrzybkowski) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/visitdashboard/VisitDashboardActivity.java]: check if the fragment isn't already added

Comment (tmarzeion) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/visitdashboard/VisitDashboardActivity.java]: Fixed

**Comment by dkayiwa** [comment]
@AdamGrzybkowski is this ready for merging? :)

**Comment by tmarzeion** [comment]
@dkayiwa We decided that we will wait for https://github.com/openmrs/openmrs-contrib-android-client/pull/209 to be merged before this one can be applied, since some changes needs to be made in this PR, because of mutual logic connections between these two PRs.

**Pull request merged**